Chat/src/app/utils/getThreadId.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const getThreadId = (): string | null => {
```

Comment:
we should rename this to something like getExistingThreadIdFromURL and comment how this sample uses the query params in the URL to be able to send/join from invite links

Reply:
Created ADO Item:
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691383